README.md

```diff
@@ -16,9 +16,12 @@ Don't like the Achievements? [Here](https://docs.github.com/account-and-profile/
 
 | Title | Badge | Earnable? | Earned by |
 | --- | --- | --- | --- |
+Pair Extraordinaire | ![Pair Extraordinaire Badge](/images/pair-extraordinaire-default.png) | ✔️ | Coauthored commits on merged pull requests
+Quickdraw | ![Quickdraw Badge](/images/quickdraw-default.png) | ✔️ | Closed an issue within 5 minutes of opening
```

Comment:
Ah, I made a mistake, this achievement only applies to pull request.

Reply:
Turns out at the very least issues can be marked as having unlocked the achievement: https://github.com/Seddryck?achievement=quickdraw&tab=achievements

I'm waiting for a clearer example to change it in the table, because in that specific example a PR was still involved - even though the issue was opened after the PR was merged and naturally the issue not auto-closed by the PR.